circle.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 parameters:
   docker-image-tag:
     type: string
-    default: "mbgl/android-ndk-r21:769c31f1d8"
+    default: "mbgl/android-ndk-r21:latest"
```

Comment:
Maybe we don't want to use latest, but need to bump the version to get python

https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/9382/workflows/c8ea2baa-6003-4606-9a87-e6271a15af57/jobs/35266
```
pip3 install requests
python3 scripts/trigger-mobile-metrics.py
/bin/bash: pip3: command not found
```